src/java/picard/analysis/CompareMetrics.java

```diff
@@ -17,13 +17,22 @@
  * Compare two metrics files.
  */
 @CommandLineProgramProperties(
-        usage = CompareMetrics.USAGE,
-        usageShort = CompareMetrics.USAGE,
+        usage = CompareMetrics.USAGE_SUMMARY + CompareMetrics.USAGE_DETAIL,
+        usageShort = CompareMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CompareMetrics extends CommandLineProgram {
-
-    static final String USAGE = "Compare two metrics files";
+    static final String USAGE_SUMMARY =  "Compares two metrics files";
+    static final String USAGE_DETAIL = "Compares the headers of the two input metrics files that have the same structure," +
+            " but come from different underlying metric classes.  Outputs can be either equal" +
```

Comment:
Actually now that I'm thinking about it, I'm not sure what this compares. The summary makes it sound like it's comparing the metrics themselves, but the details seem to imply it only compares the headers. Do we know what is the purpose of this tool, in practice?